fly/integration/expose_pipeline_test.go

```diff
@@ -114,5 +115,87 @@ var _ = Describe("Fly CLI", func() {
 			})
 		})
 
+		Context("with a custom team", func() {
```

Comment:
May I ask you to be more descriptive in this specific scenario? Maybe instead of "with a custom team", a text like in the [unpause_pipeline_test.go](https://github.com/concourse/concourse/blob/ebced261ddb2b3c76a87ff5bff26a6f4ea3cc60c/fly/integration/unpause_pipeline_test.go#L66) or sth similar. Just to clarify that in this case, the user is targeting one team and the pipeline belongs to a different one, thus the use of --team.

Reply:
@izabelacg updated accordingly